src/button/index.js

```diff
@@ -1,19 +1,31 @@
-import ButtonPrimary from './button-primary';
-import ButtonSecondary from './button-secondary';
-import ButtonWhite from './button-white';
-import ButtonPill from './button-pill';
-import ButtonWarning from './button-warning';
-import ButtonOcean from './button-ocean';
-import ButtonDark from './button-dark';
-import LoadingIcon from './button-loading';
+import React from 'react';
+import StyledButton from './style';
+import PropTypes from 'prop-types';
+import Loading from './loading.svg';
 
-export {
-  ButtonPrimary,
-  ButtonSecondary,
-  ButtonWhite,
-  ButtonPill,
-  ButtonWarning,
-  ButtonOcean,
-  ButtonDark,
-  LoadingIcon
+const Button = props => {
```

Comment:
I would refactor this, basically removes the need for the return function. Are all the props required to be passed to StyledButton? Like children and loading?

```
const Button = ({ children, loading, ...rest }) => (
   <StyledButton {...rest} children={children} loading={loading}>
    ....
);
```
or 
```
const Button = ({ children, loading, ...props }) => (
   <StyledButton {...props}>
    ....
);
```